sw/host/opentitanlib/src/util/openocd.rs

```diff
@@ -252,6 +252,11 @@ impl JtagChain for OpenOcdJtagChain {
         if resp.contains("JTAG scan chain interrogation failed") {
             bail!(OpenOcdError::InitializeFailure(resp));
         }
+        static CONNECT_FAILED_REGEX: Lazy<Regex> =
+            Lazy::new(|| Regex::new(r"target \S+ examination failed").unwrap());
```

Comment:
Blargh, that doesn't work as intended: openocd complains with this same message if we're connecting to the LC TAP -- It takes issue with the fact that it's not actually connecting to a DM.